### PR TITLE
Refactoring stat_tools module and changing 'sf' and 'cdf' for 'pval' and 'one_minus_pval' everywhere

### PR DESCRIPTION
**Pull request by ja-che**

### CODE REVIEW

File: hidimstat/clustered_inference.py [hunk 1]
```diff
@@ -154,17 +156,21 @@ def clustered_inference(X_init, y, ward, n_clusters, train_size=0.3,
     beta_hat : ndarray, shape (n_features,) or (n_features, n_times)
         Estimated parameter vector or matrix.
 
-    sf : ndarray, shape (n_features,)
-        Survival function values of every feature.
+    pval : ndarray, shape (n_features,)
+        p-values, testing the negativity (low p-values means
```
Comment (agramfort) [inline hidimstat/clustered_inference.py:160]: what do you mean by "negativity"? rejecting the null hypothesis?

File: hidimstat/stat_tools.py [hunk 2]
```diff
@@ -178,402 +136,384 @@ def sf_from_cb(cb_min, cb_max, confidence=0.95, distrib='norm', eps=1e-14):
...
     zscore = _replace_infinity(zscore, replace_val=40, method='plus-one')
 
     return zscore
 
 
-def sf_from_pval_and_sign(pval, sign, eps=1e-14):
-    """Survival function values from p-value and parameter sign.
+def pval_from_two_sided_pval_and_sign(two_sided_pval, parameter_sign,
+                                      testing_sign, eps=1e-14):
+    """Computing one-sided p-values (for testing positivity or negativity)
```
Comment (bthirion) [inline hidimstat/stat_tools.py:347]:
```suggestion
    """Computing one-sided p-values (for testing positive or negative effect sizes)
```

**Review by bthirion — COMMENTED** [review]
Thx ! 
But please try to keep PRs focus on seprate features.

File: hidimstat/clustered_inference.py (hunk 1 above)
Comment (ja-che) [reply hidimstat/clustered_inference.py:160]: rejecting negativity means the effect is positive.

Comment (bthirion) [reply hidimstat/clustered_inference.py:160]: In this context, "rejecting" stands for "rejecting the null hypothesis", hence you cannot says "rejecting negativity".

Comment (ja-che) [reply hidimstat/clustered_inference.py:160]: In general you reject an Hypothesis like H0: param <= 0 vs H1: param > 0. That was for this reason I was using this terminology but I will make it more straight forward as requested ! ;)

**Comment by ja-che** [comment]
@bthirion are you happy with the new wording of the doc string ? Thanks

File: hidimstat/clustered_inference.py [hunk 3]
```diff
@@ -157,20 +157,16 @@ def clustered_inference(X_init, y, ward, n_clusters, train_size=0.3,
...
 
     pval_corr : ndarray, shape (n_features,)
-        Corrected p-values, testing the negativity (low p-values means
-        that we may reject negativity).
+        Corrected p-values, low p-values characterize positive effect sizes.
 
     one_minus_pval : ndarray, shape (n_features,)
-        p-values, testing the positivity (low p-values means
-        that we may reject positivity).
+        One-sided p-values, low p-values characterize negative effect sizes.
```
Comment (bthirion) [inline hidimstat/clustered_inference.py:166]: Sorry this one is still misleading.
Let me propose:
one minus the p-value, with numerically accurate values, for negative effects (ie pvalue close to one).

File: hidimstat/stat_tools.py [hunk 4]
```diff
@@ -66,21 +66,22 @@ def pval_from_scale(beta, scale, testing_sign, eps=1e-14):
     scale : ndarray, shape (n_features,)
         Value of the standard deviation of the parameters.
 
-    testing_sign : {'minus', 'plus'}
-        If 'minus', testing for negativity (low p-values means we may
-        reject negativity). If 'plus', testing for positivity (low p-values
-        means we may reject positivity).
+    testing_sign : {'plus', 'minus'}
```
Comment (bthirion) [inline hidimstat/stat_tools.py:69]: Not sure what you want to do.
Normally a user could want to get a one-sided p-value or a two-sided one.
If he wants to have p-values for the opposite statistic, he tests the opposite hypothesis.

**Review by bthirion — APPROVED** [review]
Good with me.

**Pull request merged**